src/components/team-bio-card/team-bio-card.scss

```diff
@@ -0,0 +1,56 @@
+@use '../../app/globals.scss' as *;
+
+.member-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 16px;
+  max-width: 274px;
+  height: 472px;
```

Comment:
```suggestion
  height: 472px;
  
  @include mobile {
    height: 400px;
  }
```

Reducing the gap on mobile when it's a single column

### Before

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/bf311c87-3e3c-4e10-9149-2cd9e1fe4a98)

### After

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/d8bd87c0-87bc-4504-9fc9-ea58b4decb06)